charts/webapi/templates/deployment.yaml

```diff
@@ -62,8 +62,13 @@ spec:
               value: http://{{ $release }}-plr-intake:8080/api
             - name: ASPNETCORE_ENVIRONMENT
               value: {{ .Values.aspnetcore.environment }}
+            - name: RabbitMQ__Password
+              valueFrom:
+                secretKeyRef:
+                  name: rabbitmq-password
+                  key: password
             - name: RabbitMQ__HostAddress
-              value: {{ .Values.rabbitmq.host }}{{ $release }}
```

Comment:
this should work find for dev/test/prod.  There will be issues with having it work with the docker-compose though...which the developers actively use. So you'll need to formulate a plan to support this value in docker-compose environment.